include/RAJA/ListSegment.hxx

```diff
@@ -219,7 +220,17 @@ ListSegment::ListSegment(const T& indx)
   m_indx(0), m_indx_own(Unowned), m_len( indx.size() )
 {
    if ( !indx.empty() ) {
+#if defined(RAJA_USE_CUDA)
+      if ( cudaMallocManaged((void **)&m_indx, m_len*sizeof(Index_type),
```

Comment:
Generally when we use this idiom, we're using a cudaErrchk type of call, why not here?